plugins/net_plugin/net_plugin.cpp

```diff
@@ -656,6 +656,7 @@ namespace eosio {
       block_id_type                    fork_head;
       uint32_t                         fork_head_num{0};
       fc::time_point                   last_close;
+      fc::time_point                   last_dropped_trx_msg_time;
```

Comment:
This section of variables are protected by `conn_mtx`, see comment on line 652. I would move it up under `trx_in_progress_size`. It is only accessed via the connection strand, so no mutex is needed.